website/content/api-docs/auth/kubernetes.mdx

```diff
@@ -30,8 +30,10 @@ access the Kubernetes API.
 ### Parameters
 
 - `kubernetes_host` `(string: <required>)` - Host must be a host string, a host:port pair, or a URL to the base of the Kubernetes API server.
-- `kubernetes_ca_cert` `(string: "")` - PEM encoded CA cert for use by the TLS client used to talk with the Kubernetes API. NOTE: Every line must end with a newline: `\n`
+- `kubernetes_ca_cert` `(string: "")` - Optional PEM encoded CA cert for use by the TLS client used to talk with the Kubernetes API.
+  NOTE: Every line must end with a newline: `\n`
```

Comment:
```suggestion
- `kubernetes_ca_cert` `(string: "")` - Optional PEM encoded CA cert. The TLS
  client can use the cert to talk with the Kubernetes API. **Every line in the
  cert file must end with a newline character (`\n`) rather than a carriage
  return (`\r`)**.
```
Added an assumed clarification re: newline vs carriage return, but it prompted a question: do we read CRLF as ending in a newline (`\r\n`)? Or must it be explicitly LF (`\n`)?
Style correction: active voice

Reply:
I've tested locally. Only it must be explicitly LF (\n).

Wondering if we should preserve the old NOTE's content and drop "rather than a carriage return (`\r`)" for simplicity.